torch/onnx/symbolic_opset9.py

```diff
@@ -1263,10 +1263,10 @@ def slice(g, self, dim, start, end, step):
         _unimplemented("slice", "step!=1 is currently not supported")
     if start.node().kind() != 'onnx::Constant' or \
             end.node().kind() != 'onnx::Constant' or dim.node().kind() != 'onnx::Constant':
-        start_unsqueezed = g.op("Unsqueeze", start, axes_i=[0])
-        end_unsqueezed = g.op("Unsqueeze", end, axes_i=[0])
-        dim_unsqueezed = g.op("Unsqueeze", dim, axes_i=[0])
-        return g.op("DynamicSlice", self, start_unsqueezed, end_unsqueezed, dim_unsqueezed)
+        if sym_help._operator_export_type == torch.onnx.OperatorExportTypes.ONNX:
```

Comment:
Can we still keep the export to DynamicSymbolic? I think it won't hurt anything right?

Reply:
This is not helping in any case. ORT no longer has support for this op, and no other backend supports this.

Reply:
Caffe2 should support it and if it's ONNX_ATEN_FALLBACK, what do we expect here?

Reply:
Prior to the changes in this PR for applySlice, all inputs of Slice op were exported as onnx::Constant. This means that we did not really have a dynamic was of exporting Slice. With this change, onnx::Slice opset 9 does not support dynamic inputs. Also, the experimental DynamicSlice onnx op is removed, and is not supported.
But same condition holds for ATEN_FALLBACK export type. It cannot support dynamic inputs. I can remove the if statement there.